### ISSUE TITLE
Filters issue (pages.set({filters: [Object]|}) = result 0.

### ISSUE THREAD

**Issue by master7xx** [alethes/meteor-pages#198]
```
var zz = {}
> pages.filters
Object {categoryId: "14279"}
> zz = pages.filters
Object {categoryId: "14279"}
>zz.counts = 1
1
>zz
Object {categoryId: "14279", counts: 1}
>pages.set({filters: zz})
0
> pages.filters
Object {categoryId: "14279", counts: 1}

```
As a result: page not refreshed and nothing changed.
Or i miss something, i just want modify filter on fly, add and remove expressions in existing filter.

**Comment by elliotberry** [comment]
It looks like client-side filtering broke again. Can anyone confirm?

**Comment by Hizoul** [comment]
Yes I can confirm that client-side Filtering is currently broken. 
Using Meteor 1.2.1 and alethes:pages @1.8.4 everything is working fine. But using Meteor 1.3.1 and alethes:pages@1.8.6 breaks client-side filtering. On Some collections with very easy filtering it sometimes works, but on another Collection its completely broken.
If i don't set any filters then it will show nothing. Setting a Filter will add the newly filtered Items to the list. Changing the Filter again will leave the previously added Items in the List and add the new Items of the new Filter result.

**Comment by johannbotha** [comment]
+1 
client filtering is beaks one 1.3.1 but works on 1.2.1
